### DISCUSSION TITLE
browser.Load or devtoolcontext.GotoAsync ?

### DISCUSSION THREAD

**Discussion by ghost** [cefsharp/CefSharp.Dom#52] [Q&A] [answered]
Greetings,

Working with and learning about CefSharp and CefSharp.DOM. When using a DevToolContext for element location is there a good reason to choose  devtoolcontext.GotoAsync instead of browser.Load ?

**Comment by amaitland** [comment, marked as answer]
Load initiates a navigation, it doesn't wait for the page to actually load.

Use GotoAsync over LoadUrlAsync as it ties into the CDP events, and that's what the tests use.
